tests/lib/rules/indent.js

```diff
@@ -2027,6 +2027,48 @@ ruleTester.run("indent", rule, {
             options: [2]
         },
         {
+            code: unIndent`
+                const {
+                  a
+                }
+                =
+                {
+                  a: 1
+                }
+            `,
+            options: [2]
+        },
+        {
+            code: unIndent`
+                const {
+                  a
+                } = {
+                    a: 1
+                  }
```

Comment:
I don't think this case should be considered valid. If possible, we should enforce only one valid indentation for any given code. Can you move this to the `invalid` section?